bin/stencil-download.js

```diff
@@ -25,21 +26,24 @@ const options = {
     exclude: ['parsed', 'manifest.json', ...extraExclude],
     apiHost: cliOptions.host,
     channelId: cliOptions.channel_id,
+    overwrite: cliOptions.overwrite,
```

Comment:
If this is just a boolean, I would prefer more specific name, like `skipOverwriteQuestion` or `omitOverwriteAsk`.

Reply:
So this one isn't actually saying whether to ask the question, but is the actual answer to the question: Should we overwrite? Hence why it seems correct for it to just be called overwrite

Reply:
I have refactored this to simply be a boolean, rather than a "Yes" or "No" string value